ui/client/components/graph/node-modal/editors/expression/DateTimeEditor/Editor.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {useState, useEffect} from "react"
+import {ExpressionObj} from "../types"
+import {useTranslation} from "react-i18next"
+import DateTimePicker from "react-datetime"
+import "react-datetime/css/react-datetime.css"
+import classNames from "classnames"
+import {useDebouncedCallback} from "use-debounce"
+import moment from "moment"
+import {asLocalDateString} from "./DateEditor"
+import {asLocalTimeString} from "./TimeEditor"
+import {asLocalDateTimeString} from "./DateTimeEditor"
+
+/* eslint-disable i18next/no-literal-string */
+export enum JavaTimeTypes {
+  LOCAL_DATE = "LocalDate",
+  LOCAL_DATE_TIME = "LocalDateTime",
+  LOCAL_TIME = "LocalTime",
+}
+
+/* eslint-enable i18next/no-literal-string */
+
+export type EditorProps = {
+  expressionObj: ExpressionObj,
+  readOnly: boolean,
+  className: string,
+  onValueChange,
+  expressionType: JavaTimeTypes,
+  dateFormat: string,
+  timeFormat?: string,
+}
+
+const parse = ({expression}: ExpressionObj, expressionType: JavaTimeTypes): moment.Moment => {
+  const [e, type, date] = /^T\(java\.time\.(.*)\)\.parse\([\'\"](.*)[\'\"]\)$/.exec(expression) || []
```

Comment:
Can we make it somewhat configurable? In many places we use util functions? @arkadius did you have any ideas about this?

Reply:
We can return this config at settings api.

Reply:
I agree that it should be in one place. But IMO it should be in some utils class. We don't need it on backend side for now.